board/early_init.h

```diff
@@ -6,10 +6,17 @@
 extern void *g_pfnVectors;
 extern uint32_t enter_bootloader_mode;
 
+typedef void (*bootloader_fcn)(void);
+typedef bootloader_fcn *bootloader_fcn_ptr;
+
 void jump_to_bootloader(void) {
   // do enter bootloader
   enter_bootloader_mode = 0;
-  void (*bootloader)(void) = (void (*)(void)) (*((uint32_t *)BOOTLOADER_ADDRESS));
+
+  // retrieve bootloader function pointer that is stored at BOOTLOADER_ADDRESS
```

Comment:
remove the comments. comments that are code rewritten as english don't tend to add anything